### PR TITLE
framework: disable some of the verbose log statements

### PR DESCRIPTION
**Pull request by liaxim**

### CODE REVIEW

File: GVRf/Framework/framework/src/main/jni/gl/gl_index_buffer.cpp [hunk 1]
```diff
@@ -7,6 +7,10 @@
 #include "gl_index_buffer.h"
 #include "gl_shader.h"
 
+//comment out to enable verbose logging
+#undef LOGV
+#define LOGV
```
Comment (dnvasilev) [inline GVRf/Framework/framework/src/main/jni/gl/gl_index_buffer.cpp]: Clever, but:
LOGV(x,y,z) becomes (x,y,z) -- meaning all garbage strings, side effects, etc. are still in the code.
I would rather do varargs macro. Does anyone remember how varargs macros work in C preprocessor?

Comment (dnvasilev) [reply GVRf/Framework/framework/src/main/jni/gl/gl_index_buffer.cpp]: #define LOGV(...) /* nothing */

Comment (liaxim) [reply GVRf/Framework/framework/src/main/jni/gl/gl_index_buffer.cpp]: The compiler takes care of it. Strings are nowhere to be found.

Now if someone has side-effects in a trace statement, it is theirs problem.

Comment (liaxim) [reply GVRf/Framework/framework/src/main/jni/gl/gl_index_buffer.cpp]: But if changing ``#define LOGV`` to ``#define LOGV(...)`` will make you happier, I will make the change.

**Comment by liaxim** [comment]
If there are no other comments, planning to merge on Wednesday

File: GVRf/Framework/framework/src/main/jni/gl/gl_index_buffer.cpp (hunk 1 above)
Comment (dnvasilev) [reply GVRf/Framework/framework/src/main/jni/gl/gl_index_buffer.cpp]: Better idea. Kill LOGV in gvr_log.h

Comment (liaxim) [reply GVRf/Framework/framework/src/main/jni/gl/gl_index_buffer.cpp]: Can be quite useful to turn on and off verbose logging for the individual units. As long as those are well organized of course.

Comment (liaxim) [reply GVRf/Framework/framework/src/main/jni/gl/gl_index_buffer.cpp]: And also LOGV should be used appropriately for what truly is verbose logging. I am not convinced it is throughout the codebase so I rather deal with individual cases for now. I am even seeing LOGE for cases where LOGI should've been used so..

But if you insist, I will turn it off globally and be done with it.

Comment (dnvasilev) [reply GVRf/Framework/framework/src/main/jni/gl/gl_index_buffer.cpp]: This is new functionality and I would organize it a bit differently, without undef

Comment (dnvasilev) [reply GVRf/Framework/framework/src/main/jni/gl/gl_index_buffer.cpp]: Makes sense for now to turn it off globally.

Comment (dnvasilev) [reply GVRf/Framework/framework/src/main/jni/gl/gl_index_buffer.cpp]: If we allow verbose logging to impact performance, it must be turned off for release

**Pull request closed without merging**